docs/platform-engineers/system-operation/offline-installation.md

```diff
@@ -0,0 +1,58 @@
+# KubeVela Offline Installation Guide
+
+KubeVela 离线部署包含 KubeVela Core 和 KubeVela Addon 的离线部署。
+
+## KubeVela Core 离线部署
+
+### KubeVela chart
+
+- 下载 vela-core Helm Chart 包
+
+通过 [Helm Chart 安装 KubeVela Core](http://kubevela.net/docs/install)文档下载您希望的 vela-core Chart 包，并解压。
+
+- 修改 values 里可配置的镜像
+
+拉取以下镜像并导入离线环境的镜像仓库， 在 `helm install` 安装命令里通过 `--set` 覆盖每个镜像对应的离线环境镜像仓库里每个镜像的信息。
+
+```shell
+$  kubevela git:(master) grep -r repository charts/vela-core/values.yaml -A 1
+charts/vela-core/values.yaml:  repository: oamdev/vela-core
+charts/vela-core/values.yaml-  tag: latest
+--
+charts/vela-core/values.yaml:      repository: oamdev/kube-webhook-certgen
+charts/vela-core/values.yaml-      tag: v2.3
+--
+charts/vela-core/values.yaml:      repository: oamdev/cluster-gateway
+charts/vela-core/values.yaml-      tag: v1.1.7
+--
+charts/vela-core/values.yaml:    repository: oamdev/hello-world
+charts/vela-core/values.yaml-    tag: v1
+--
+charts/vela-core/values.yaml:    repository: oamdev/alpine-k8s
+charts/vela-core/values.yaml-    tag: 1.18.2
+```
```

Comment:
we have a values replace way by setting like below:

```
helm install ....  --set imageRegistry=kubevela-registry.cn-hangzhou.cr.aliyuncs.com/
```

Refer to this PR: https://github.com/oam-dev/kubevela/pull/2623

But this requires the user to upload the images to their private image repo in a unified way.

Reply:
Yes, that's what I mean for the configurable images.